common/src/main/java/com/lightstep/tracer/shared/Options.java

```diff
@@ -442,7 +442,7 @@ private void defaultComponentName() {
                     if (st.hasMoreTokens()) {
                         String name = st.nextToken();
                         withComponentName(name);
-                        tags.put(LEGACY_COMPONENT_NAME_KEY, name);
```

Comment:
Does this need to get called out in the CHANGELOG? Is this is backward incompatible in some way? It would be really good to be clear about that so it's easy for us to update documentation.